From 1e6b9c6eab6f569579d66cf2d66d9c6358c5adef Mon Sep 17 00:00:00 2001
From: Guchun Chen <guchunc@codeaurora.org>
Date: Thu, 13 Oct 2016 11:08:41 +0800
Subject: [PATCH] drm : msm : add modifier support in kms driver.

In kms driver init process, allow_fb_modifiers should be set ture
to support source compressed data.

Change-Id: I4527c7413fdb2fcc1e5c17b40f10c6137e491831
Signed-off-by: Guchun Chen <guchunc@codeaurora.org>
---
 drivers/gpu/drm/msm/mdp/mdp5/mdp5_kms.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/msm/mdp/mdp5/mdp5_kms.c b/drivers/gpu/drm/msm/mdp/mdp5/mdp5_kms.c
index 80e5af8..5a9c0fc 100644
--- a/drivers/gpu/drm/msm/mdp/mdp5/mdp5_kms.c
+++ b/drivers/gpu/drm/msm/mdp/mdp5/mdp5_kms.c
@@ -723,6 +723,8 @@ struct msm_kms *mdp5_kms_init(struct drm_device *dev)
 	dev->mode_config.max_width = config->hw->lm.max_width;
 	dev->mode_config.max_height = config->hw->lm.max_height;
 
+	dev->mode_config.allow_fb_modifiers = true;
+
 	return kms;
 
 fail:
-- 
1.9.1

